forte/data/data_store.py

```diff
@@ -106,8 +109,9 @@ def __init__(self):
         self.elements: List = []
```

Comment:
Same, should make them private with two dash `__elements`

Reply:
make sense